SwrveSDK/src/main/java/com/swrve/sdk/messaging/SwrveCampaignState.java

```diff
@@ -0,0 +1,61 @@
+package com.swrve.sdk.messaging;
+
+import com.swrve.sdk.SwrveLogger;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/**
+ * Used internally to save the state of campaigns
+ */
+public class SwrveCampaignState {
+    protected static final String LOG_TAG = "SwrveMessagingSDK";
+
+    // Number of impressions of this campaign. Used to disable the campaign if
+    // it reaches total impressions
+    protected int impressions;
+
+    // Inbox status of the campaign
+    protected SwrveCampaignStatus status;
+
+    // Next message to be shown if round robin campaign
+    public int next;
+
+    public SwrveCampaignState() {
+        impressions = 0;
+        status = SwrveCampaignStatus.Unseen;
+        next = 0;
+    }
+
```

Comment:
One is loading, another one is creating one from scratch (first time the campaign is received)